tools/automatic_query_fixer/src/test/java/com/google/cloud/bigquery/utils/queryfixer/QueryTokenProcessorTest.java

```diff
@@ -15,16 +16,47 @@
 
   @Before
   public void createService() {
-    tokenService = new QueryTokenProcessor(new BigQueryParserFactory());
+    tokenService = new QueryTokenProcessor(new CalciteTokenizer(new BigQueryParserFactory()));
   }
 
   @Test
   public void convertQueryToTokens() {
-    String sql = "Select col from `d1.t1`\n" + "where t1.col>'val'";
+    String query = "Select col from `d1.t1`\n" + "where t1.col>'val'";
+
+    List<IToken> tokens = tokenService.getAllTokens(query);
+    QueryPositionConverter converter = new QueryPositionConverter(query);
 
-    List<IToken> tokens = tokenService.getAllTokens(sql);
-    assertEquals(10, tokens.size());
-    assertEquals("Select", tokens.get(0).getImage());
-    assertEquals("'val'", tokens.get(tokens.size() - 1).getImage());
+    for (IToken token : tokens) {
+      int startIndex = converter.posToIndex(token.getBeginRow(), token.getBeginColumn());
+      int endIndex = converter.posToIndex(token.getEndRow(), token.getEndColumn() + 1);
+      assertEquals(
+          token.getImage().toUpperCase(), query.substring(startIndex, endIndex).toUpperCase());
+    }
+  }
+
+  @Test
+  public void verifyModifiedQuery() {
+    String origin = "Select col from t1 Join\nt2 on t1.id = t2.id\nwhere t1.col > 'val'";
+    String target;
+
+    target = "Select Distinct col from t1 Join\nt2 on t1.id = t2.id\nwhere t1.col > 'val'";
```

Comment:
merged into one line? String target = "...."

Reply:
Yes, the auto formatter put them in one line with `+`s, so I deleted the `+`s since they are in one line.